test/widgets/app_test.dart

```diff
@@ -319,6 +351,27 @@ void main() {
         ..page.isA<HomePage>();
     });
 
+    testWidgets('choosing an account changes the last visited account', (tester) async {
+      addTearDown(testBinding.reset);
+      await testBinding.globalStore.add(eg.selfAccount, eg.initialSnapshot());
+      await testBinding.globalStore.add(
+        eg.otherAccount, eg.initialSnapshot(realmUsers: [eg.otherUser]),
+        markLastVisited: false);
```

Comment:
nit: can simplify to not use `markLastVisited`:
```suggestion
      await testBinding.globalStore.add(
        eg.otherAccount, eg.initialSnapshot(realmUsers: [eg.otherUser]));
```

and just flip around which account is which in the later steps below.

(For existing test cases, it's good to use `markLastVisited` wherever that makes it possible to avoid other changes to the tests. But when adding a new test, it's just as easy to add the alternative version as to add this version.)